__tests__/app.test.js

```diff
@@ -62,4 +62,53 @@ describe("/api", () => {
             });
     });
 
+})
+describe("/api/articles/:article_id", () => {
+    test("GET 200: Responds with endpoint json data", () => {
+        return request(app)
+            .get("/api/articles/4")
+            .expect(200)
+            .then(({ body }) => {
+              const article = body.article;
+              article.forEach((article) => {
+                  expect(typeof article.article_id).toBe("number");
+                  expect(typeof article.title).toBe("string");
+                  expect(typeof article.topic).toBe("string");
+                  expect(typeof article.author).toBe("string");
+                  expect(typeof article.body).toBe("string");
+                  expect(typeof article.created_at).toBe("string");
+                  expect(typeof article.votes).toBe("number");
+                  expect(typeof article.article_img_url).toBe("string");
+                //do I include a test for definite values?
```

Comment:
Great to see you thinking about this Alec! Yes, I'd argue that it makes most sense for this endpoint for us to assert the exact values expected on our response. We know that if a user requests article 4 then they should get back only the information relevant to that article. In doing this more thorough check though, we no longer have need for the previous assertions where we are using expect .any... etc.